docs/src/pages/guides/scripts.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Scripts
+---
+
+Astro automatically adds script tags for your partially hydrated components and bundles those in production.
+
+Any other script that you want to run in your page you'll add it by adding a `<script>` somewhere in your page, or within a component. 
+
+# Hoisted scripts
```

Comment:
I would recommend to move this section into `astro-components.md`, similar to how `### CSS Styles`  exists as a section in that doc.

Reply:
Moved this